### PR TITLE
Added swagger-self-hosted.md

### PR DESCRIPTION
**Pull request by jilleJr**
> For Iver employees: WO0000000681896

Original docs from private repo was:

> ## Swagger API
>
> Go to `api/swagger/index.html` or `import/<provider>/swagger/index.html`.

I expanded it a little. Hopefully it's still comprehensible 

## Preview

https://github.com/iver-wharf/iver-wharf.github.io/blob/f443c81d5586e57fcb554f48f6aaf56cd15dd4da/docs/reference/swagger-self-hosted.md

Rendered:

image

### CODE REVIEW

File: docs/reference/swagger-self-hosted.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Swagger (self-hosted)
+
+Wharf's main API and provider APIs all produce OpenAPI specifications.
```
Comment (iverestefans) [inline docs/reference/swagger-self-hosted.md:3]: shouldn't be here provider**s** APIs?

**Review by iverestefans — APPROVED** [review]
LGTM!

File: docs/reference/swagger-self-hosted.md (hunk 1 above)
Comment (jilleJr) [reply docs/reference/swagger-self-hosted.md:3]: I've no idea. I'm used to only pluralize the last word. If that's correct or not I have no clue

**Review by vreve — APPROVED** [review]

File: docs/reference/swagger-self-hosted.md (hunk 1 above)
Comment (jilleJr) [reply docs/reference/swagger-self-hosted.md:3]: K if I resolve?

Comment (iverestefans) [reply docs/reference/swagger-self-hosted.md:3]: I am ok with it.

**Review by iverestefans — APPROVED** [review]

**Pull request merged**